setup.cfg

```diff
@@ -30,6 +30,7 @@ exclude =
     .eggs/*.py,
     versioneer.py,
     env  # exclude asv benchmark environments from linting
```

Comment:
why is this excluded? shouldn’t it just be formatted ?

Reply:
The whole module generates errors about unused imports for anything internal since we import them but then only use a string literal as a forward reference.

We could alternately add `noqa: F401` at the end of each internal import